core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -616,7 +616,15 @@ private ClientFactoryOptions buildOptions() {
                         if (dnsResolverGroupCustomizers != null) {
                             dnsResolverGroupCustomizers.forEach(consumer -> consumer.accept(builder));
                         }
-                        return builder.build(eventLoopGroup);
+
+                        final ClientFactoryOptionValue<?> opt = options
+                                .getOrDefault(ClientFactoryOptions.METER_REGISTRY,
+                                ClientFactoryOptions.METER_REGISTRY
```

Comment:
It's basically same. I suggested this so that we can use the default `MeterRegistry` defined in `ClientFactoryOptions`, but I'm fine with @minwoox's suggestion given our default will not change.

Reply:
I'm fine as it is. 😄  Just don't forget to indent the line. 😄

Reply:
Done.